src/Microsoft.IdentityModel.Protocols.WsFederation/WsFederationConstants.cs

```diff
@@ -145,6 +145,21 @@ public static class keyUse
             public const string Signing = "signing";
         }
 
+        /// <summary>
+        /// xmlns string.
+        /// </summary>
+        public static string Xmlns = "xmlns";
+
+        /// <summary>
+        /// Sub namespace names.
+        /// </summary>
+        public static class NamespaceName
```

Comment:
These are not really namespaces, but prefixes.
For Saml use called it preferred namespace

see: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/dev/src/Microsoft.IdentityModel.Tokens.Saml/Saml/SamlConstants.cs#L41